PerDiem/PerDiem/Controllers/CalendarTransition.m

```diff
@@ -35,17 +36,33 @@ - (void)animateTransition:(id <UIViewControllerContextTransitioning>)transitionC
     
     CalendarDayViewController *dayVc;
     CalendarViewController *monthVc;
-    
+
+    // This doesn't too nice, and also the transition I made is not consistent with the rest of the system.
+    if ([[transitionContext viewControllerForKey:UITransitionContextToViewControllerKey] isKindOfClass:[TransactionFormViewController class]] ||
+        [[transitionContext viewControllerForKey:UITransitionContextFromViewControllerKey] isKindOfClass:[TransactionFormViewController class]]) {
+        UIViewController *toViewController = [transitionContext viewControllerForKey:UITransitionContextToViewControllerKey];
+        [[transitionContext containerView] addSubview:toViewController.view];
+        toViewController.view.alpha = 0.0;
+        [UIView animateWithDuration:0.8 animations:^{
+            toViewController.view.alpha = 1.0;
+            if ([toViewController isKindOfClass:[TransactionFormViewController class]]) {
+                toViewController.navigationController.navigationBar.alpha = 1;
```

Comment:
might tinker with that indeed, but one of the reason why I was changing the navigation alpha was because the status bar would default to black color if the navigation was hidden - and for some reason alpha = 0 was making the nav bar hidden XD